packages/react-components/src/components/ChatMessage/ChatMessageContent.tsx

```diff
@@ -180,6 +183,16 @@ const extractContentForAllyMessage = (props: ChatMessageContentProps): string =>
       parsedContent.replaceChild(imageTextNode, child);
     });
 
+    // Inject attachment names for aria.
+    if (attachments) {
```

Comment:
condition compile this whole node, and use `props.message.attachments` instead of `attachments`?

Reply:
props.message doesn't have attachments in stable, so it won't work